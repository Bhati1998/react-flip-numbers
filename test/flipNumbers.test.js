import renderer from 'react-test-renderer';
import FlipNumbers from 'universal/components/flipNumbers/flipNumbers';

jest.mock('./flipNumber', () => 'FlipNumber');

describe('FlipNumbers', () => {
  const props = {
    numbersToDisplay: '00:00',
    breakCharacterClassName: ':',
    height: 20,
    width: 20,
    color: 'black',
    background: 'white',
    perspective: '1000px',
    durationSeconds: 0.3,
    delaySeconds: 0.3,
    startAnimation: false,
  };

  it('should render correctly', () => {
    const tree = renderer.create(<FlipNumbers {...props} />);
    expect(tree).toMatchSnapshot();
  });
});
