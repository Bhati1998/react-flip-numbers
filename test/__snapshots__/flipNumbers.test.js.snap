// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlipNumbers should render correctly 1`] = `
<section
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "justifyContent": "center",
    }
  }
>
  <FlipNumber
    activeNumber={0}
    background="white"
    color="black"
    delaySeconds={0.3}
    durationSeconds={0.3}
    height={20}
    length={5}
    numberStyle={Object {}}
    perspective="1000px"
    play={false}
    position={1}
    width={20}
  />
  <FlipNumber
    activeNumber={0}
    background="white"
    color="black"
    delaySeconds={0.3}
    durationSeconds={0.3}
    height={20}
    length={5}
    numberStyle={Object {}}
    perspective="1000px"
    play={false}
    position={2}
    width={20}
  />
  <span
    style=":"
  >
    :
  </span>
  <FlipNumber
    activeNumber={0}
    background="white"
    color="black"
    delaySeconds={0.3}
    durationSeconds={0.3}
    height={20}
    length={5}
    numberStyle={Object {}}
    perspective="1000px"
    play={false}
    position={4}
    width={20}
  />
  <FlipNumber
    activeNumber={0}
    background="white"
    color="black"
    delaySeconds={0.3}
    durationSeconds={0.3}
    height={20}
    length={5}
    numberStyle={Object {}}
    perspective="1000px"
    play={false}
    position={5}
    width={20}
  />
</section>
`;
